docs/marketplace/dev/glue-api-guides/202200.0/shopping-lists/managing-shopping-lists.md

```diff
@@ -0,0 +1,1127 @@
+                "metaKeywords": "Lenovo,Tax Exempt",
+                "metaDESCRIPTION": "Small Form Factor Small Form Factor desktops provide the ultimate performance with full-featured scalability, yet weigh as little as 13.2 lbs / 6 kgs. Keep",
+                "attributeNames": {
+                    "processor_threads": "Processor Threads",
+                    "pci_express_slots_version": "PCI Express slots version",
+                    "internal_memory": "Max internal memory",
+                    "stepping": "Stepping",
+                    "brand": "Brand",
+                    "processor_frequency": "Processor frequency"
+                }
+            },
+            "links": {
+                "self": "http://glue.mysprykershop.com/concrete-products/128_27314278"
+            }
+        },
+        {
+            "type": "shopping-list-items",
+            "id": "00fed212-3dc9-569f-885f-3ddca41dea08",
+            "attributes": {
+                "quantity": 1,
+                "sku": "128_27314278"
+            },
+            "links": {
+                "self": "http://glue.mysprykershop.com/shopping-lists/ecdb5c3b-8bba-5a97-8e7b-c0a5a8f8a74a/shopping-list-items/00fed212-3dc9-569f-885f-3ddca41dea08"
+            },
+            "relationships": {
+                "concrete-products": {
+                    "data": [
+                        {
+                            "type": "concrete-products",
+                            "id": "128_27314278"
+                        }
+                    ]
+                }
+            }
+        }
+    ]
+}
+```
+</details>
+
+For response attributes, see [Create a shopping list](#create-a-shopping-list).
+
+For the attributes of included resources, see:
+* [Add items to a shopping list](/docs/scos/dev/glue-api-guides/{{page.version}}/managing-shopping-lists/managing-shopping-list-items.html#shopping-list-items-response-attributes)
+* [Retrieve a concrete product](/docs/scos/dev/glue-api-guides/{{page.version}}/managing-products/concrete-products/retrieving-concrete-products.html#concrete-products-response-attributes)
+
+## Delete a shopping list
+
+To delete a shopping list, send the request:
+
+***
+`DELETE` **shopping-lists/*{% raw %}{{{% endraw %}shopping_list_id{% raw %}}}{% endraw %}***
+***
+
+| PATH PARAMETER | DESCRIPTION |
+| --- | --- |
+| ***{% raw %}{{{% endraw %}shopping_list_id{% raw %}}}{% endraw %}*** | Unique identifier of a shopping list to delete. To get it, [Retrieve shopping lists](#retrieve-shopping-lists). |
+
+### Request
+
+Request sample: 
+
+`DELETE http://glue.mysprykershop.com/shopping-lists/sdb17f85-953f-565a-a4ce-e5cb02405f83` — Delete the shopping list with the id `sdb17f85-953f-565a-a4ce-e5cb02405f83`.
+
+### Response
+
+If the shopping list is deleted successfully, the endpoint returns the `204 No Content` status code.
+
+## Possible errors
+
+| CODE | REASON |
+| --- | --- |
+| 001 | Access token is incorrect. |
```

Comment:
Some of the error codes are missing https://spryker.atlassian.net/wiki/spaces/CORE/pages/226427002/Error+codes#Errorcodes-ShoppingLists